demos/llm_npu/README.md

```diff
@@ -38,7 +38,7 @@ Run `export_model.py` script to download and quantize the model:
 
 **LLM**
 ```console
-python export_model.py text_generation --source_model mistralai/Mistral-7B-Instruct-v0.2 --target_device NPU --config_file_path models/config.json --model_repository_path models  --overwrite_models
+python export_model.py text_generation --source_model meta-llama/Llama-3.1-8B-Instruct --target_device NPU --config_file_path models/config.json --model_repository_path models  --overwrite_models
```

Comment:
No need for precision reduction? If it happens implicitly when choosing NPU as target device, I think we should point it out in the readme

Reply:
@mzegla Precision is set to INT4 automatically and info about changes in precision is printed in the output.